modin/_version.py

```diff
@@ -96,9 +112,10 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
         if verbose:
             print("unable to find command, tried %s" % (commands,))
         return None, None
-    stdout = p.communicate()[0].strip()
+    stdout: str
+    stdout = p.communicate()[0].strip()  # type: ignore
```

Comment:
Why ignore?

Reply:
This ignore shouldn't be here; I didn't realize `bytes` also had a `strip` method (`stdout` should be annotated as `bytes` instead).